rollup.config.js

```diff
@@ -30,8 +30,7 @@ const onwarn = (warning) => {
 
 const primedIgnore = ignore(['videojs-vtt.js']);
 const primedResolve = resolve({
-  jsnext: true,
-  main: true,
+  mainFields: ['module', 'main'],
```

Comment:
should probably add `jsnext` back in here for backwards compatibility, although I doubt that it was used.